app/services/data_migrations/backfill_chase_provider_decision_setting.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
+  class BackfillChaseProviderDecisionSetting
+    TIMESTAMP = 20220225092605
+    MANUAL_RUN = false
+
+    def change
+      ProviderUserNotificationPreferences.where(application_received: false).update_all(chase_provider_decision: false)
```

Comment:
could we use `update` to ensure callbacks, as this will make sure BQ is updated correctly with those values, I think for application choices it is a good idea to avoid touching the application but here there is no risk of that

Reply:
I've changed it, it's now using the `update!` method instead.